bundle/edu.gemini.itc/src/main/java/edu/gemini/itc/gmos/Gmos.java

```diff
@@ -296,6 +296,23 @@ private void validate() {
                 "Please use the ITC in 1-slit mode to estimate sensitivity, and see the GMOS IFU web pages for a discussion of spectral overlap.");
             }
 
+            // central wavelength, site dependent
+            double _centralWavelength = getCentralWavelength();
+            switch (gp.site()) {
+                // User-input central wavelength for GN
+                case GN:
+                    if (_centralWavelength < 360 || _centralWavelength > 1000)
+                        throw new RuntimeException("Central wavelength must be between 360 nm and 1000 nm.");
+                    break;
```

Comment:
Seems like the issue is that `sbt` isn't found. I would imagine re-running with @cquiroz's earlier fixes might solve the problem?